packages/core/src/App/Components/Layout/Header/toggle-menu-drawer.jsx

```diff
@@ -562,6 +566,7 @@ const ToggleMenuDrawer = React.forwardRef(
                                                 onClick={() => {
                                                     setIsPreAppStore(true);
                                                     toggleDrawer();
+                                                    history.push(routes.trading_hub);
```

Comment:
These three code blocks are repeated. Can extract to a function?

Reply:
Sure 👍